next/components/Molecules/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,108 @@
+import cx from 'classnames'
+import React, { useRef } from 'react'
+
+import SliderControl, { SliderVariant } from './SliderControl/SliderControl'
+
+interface SectionPromosProps {
+  listClassName?: string
+  itemClassName?: string
+  items: { element: React.ReactNode | null; key: string | undefined }[]
```

Comment:
ReactNode includes `null` so its redundant to write `| null` :) 

We also prefer to import `ReactNode` and use it directly `element: ReactNode`. (Similarly `useEffect`, `useMemo`...)